vite.config.ts

```diff
@@ -13,7 +28,8 @@ export default defineConfig({
   },
   resolve: {
     alias: {
-      lodash: 'lodash-es'
+      lodash: isTest ? 'lodash' : 'lodash-es',
```

Comment:
Is this needed?

Reply:
`@testing-library/jest-dom` installs `lodash`, can't hurt to keep it.